client/src/pages/Signup.js

```diff
@@ -40,8 +40,8 @@ const Signup = () => {
 
     return (
         <Grid container className={classes.vh100}>
```

Comment:
as the screen resizes, the container changes to a vertical view from a horizontal view, can you try using flex direction or display inline-block to tackle it

Reply:
you mean to change the Grid into flex/inline-bloack??

Reply:
I dont think this is a bug. I think all react apps work this way. It makes sense for it to switch from horizontal to vertical. My thinking is where would the image go if the container did not switch from horizontal to vertical? There is no other space on  the screen, like on a phone.

Reply:
Yes,  I agree. It should turn to vertical on small screen